django_inlinecss/conf.py

```diff
@@ -1,9 +1,3 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-from __future__ import unicode_literals
-
-
 try:
     import importlib
 except ImportError:
```

Comment:
In python 3 just `import importlib` .  
There is no `from django.utils import importlib` in django 2.2